packages/vue-component-library/src/stories/ImageSlider.stories.js

```diff
@@ -86,3 +87,43 @@ export function WithCustomLabels() {
     `,
   }
 }
+
+export function WithScrollWrapperCarousel() {
+  return {
+    data() {
+      return {
+        ...mockBeforeAfterImageCarousel[0],
+      }
+    },
+    components: { ImageSlider, ScrollWrapper },
+    template: `
+        <component is="style" type="text/css">
+.before-image {
+    filter: grayscale(100%)
+}
+    </component>
+        <div style="padding: 1rem; max-width: 1160px;">
+          <ScrollWrapper :single-item="true" :initial-slide="1">
```

Comment:
@farosFreed Shouldn't `:single-item` be `false` since there are multiple sliders getting passed to the ScrollWrapper?